docs/custom_conf.py

```diff
@@ -146,6 +146,24 @@
 
 ## The following settings override the default configuration.
 
+manpages_url = 'https://manpages.ubuntu.com/manpages/noble/en/man{section}/{page}.{section}.html'
```

Comment:
Can we use the version-agnostic URL instead? 

Should be something like https://manpages.ubuntu.com/manpages/en/man{section}/{page}.{section}.html I think - iirc it defaults to the most recent release, but otherwise we'll have to remember to update this every 6 months

Reply:
Didn't knew that this redirection exist.